config.py

```diff
@@ -78,18 +78,42 @@
 
 # Social Links
 
-social_links: list[dict[str, str | bool]] = [
-    {"name": "Website", "url": "https://revanced.app", "preferred": True},
-    {"name": "GitHub", "url": "https://github.com/revanced", "preferred": False},
-    {"name": "Twitter", "url": "https://twitter.com/revancedapp", "preferred": False},
-    {"name": "Discord", "url": "https://revanced.app/discord", "preferred": True},
+social_links: list[dict[str, str, str, str | bool]] = [
+    {
+        "name": "Website",
+        "url": "https://revanced.app",
+        "preferred": True,
+    },
+    {
+        "name": "Discord",
+        "url": "https://discord.gg/revanced",
```

Comment:
The reason this link was not chosen is because the /revanced invite is Nitro boosts dependant. Discord has this feature where Nitro subscribers can boost a Discord server. If the boost is lost after some time, so is that invite, and only when people boost the server again, the link can be reclaimed, if not done by someone else. For that reason we have made our own link at revanced.app/discord in case we lose an invite for any reason

```suggestion
        "url": "https://revanced.app/discord",
```

Reply:
Manager should use the API in the first place, yes